_posts/2024-06-24-tenant-admin.md

```diff
@@ -0,0 +1,82 @@
+- Image Credit: The credit for the image that will be displayed on the hero banner for this tenant. This should contain the name of the person or organization that created the image.
+- Image Description: An _accessible_ description of the image. This will not be displayed, but will be used to describe the image for users who are using screen readers or other assistive technologies.
+
+![A screenshot of the MET app. The bottom of the "Create Tenant" form is displayed. The visible portion of the form includes an optional field for "Image description", which is empty, and a "Create Instance" button, outlined in red.](/assets/UserGuideImages/Images/tenant-admin/tenant-admin-add-tenant-form-bottom.png){: .light .w-75 .shadow .rounded-10}
+
+## Edit Tenant
+
+To edit a tenant, follow the steps in ["View Tenant Details"](#view-tenant-details) to navigate to the details page of the tenant you wish to edit. From there, press the **Edit** button to open the edit form.
+
+![A screenshot of the MET app. The top of the tenant details page for Government Digital Experience is displayed. The "Edit" button is outlined in red.](/assets/UserGuideImages/Images/tenant-admin/tenant-admin-edit-tenant-button.png){: .light .w-75 .shadow .rounded-10}
+
+This will open a form where you can edit the tenant's details. The information you need to fill out is the same as when [creating a tenant](#create-tenant). It is fine to leave some fields unchanged, but at least one edit must be made before you can save. Once you have made your desired changes, select the **Update Instance** button to save them.
+
+![A screenshot of the MET app. The bottom of the "Edit Tenant" form is displayed. The visible portion of the form includes an uploaded image, a field for "Image Credit" which has the value "NASA", an "Image Description" field which has the value "A galaxy with stars. In the centre is a bright swirl of dust and gas", and an "Update Instance" button, outlined in red.](/assets/UserGuideImages/Images/tenant-admin/tenant-admin-edit-tenant-form-bottom.png){: .light .w-75 .shadow .rounded-10}
+
+## Delete Tenant
+
+To delete a tenant, follow the steps in ["View Tenant Details"](#view-tenant-details) to navigate to the details page of the tenant you wish to delete. From there, press the **Delete** button to open the delete confirmation modal.
+
+![A screenshot of the MET app. The top of the tenant details page for Government Digital Experience is displayed. The "Delete Tenant Instance" button has an icon of a trash can and is outlined in red.](/assets/UserGuideImages/Images/tenant-admin/tenant-admin-delete-tenant-button.png){: .light .w-75 .shadow .rounded-10}
+
+In the modal, you will be asked to confirm that you want to delete the tenant. If you are sure you want to proceed, select the **Delete Instance** button. This action is irreversible, so please be certain before proceeding.
```

Comment:
I know there are still some things to sort out when it comes to deleting tenants, but I think safe to assume that all associated Engagments, Surveys, Widgets will go too right? I think we could inform the user of that here so they understand the stakes. The image says "all associated info will be deleted" but the user might assume that just means the hero image, short name, etc.